File: 2da parte/flutter/flutter-end/lib/model/main_weather_model.dart

import 'package:TallerUNAJ2/model/weather_model.dart';

import 'temp_model.dart';

class MainWeatherModel{
  String cityName;
  WeatherModel description;
  TempModel temperature;

  MainWeatherModel(this.cityName, this.description, this.temperature);
}

File: 3ra parte/kotlin/kotlin-start/app/src/main/java/com/unaj/tallerunajweatherapp/model/MainWeatherModel.kt
package com.unaj.tallerunajweatherapp.view.model

import android.os.Parcelable
import com.unaj.tallerunajweatherapp.model.TempModel
import com.unaj.tallerunajweatherapp.model.WeatherModel
import kotlinx.android.parcel.Parcelize

@Parcelize
data class MainWeatherModel (
    val cityName: String,
    val description: List<WeatherModel>,
    var temperature: TempModel
): Parcelable
